File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/toasts/ToastsActivity.java
package com.jdroid.android.sample.ui.toasts;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class ToastsActivity extends FragmentContainerActivity {

	/**
	 * @see com.jdroid.android.activity.FragmentContainerActivity#getFragmentClass()
	 */
	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return ToastsFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/toasts/ToastsActivity.kt
package com.jdroid.android.sample.ui.toasts

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class ToastsActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return ToastsFragment::class.java
    }
}
